autoload/quickpick.vim

```diff
@@ -187,6 +187,10 @@ function! quickpick#busy(busy) abort
   endif
 endfunction
 
+function! quickpick#results_winid() abort
+  return s:state['resultswinid']
```

Comment:
`s:state` can be undefined before calling it or when it is closed by `unlet s:state`